ui/app/routes/workspace/projects/project/app.ts

```diff
@@ -56,6 +56,7 @@ export default class App extends Route {
     appRef.setProject(proj.name);
 
     return hash({
+      project: proj,
```

Comment:
Can we add this field to the `Model` interface defined at the top of the file?

Reply:
done

Reply:
I'll use the `squash and merge` button on GH